blocks/additional-info/__link/additional-info__link.css

```diff
@@ -3,6 +3,13 @@
   line-height: 19px;
 }
 
+@media only screen and (max-width: 320px) {
```

Comment:
I don't have where to write it but, the additional title in 320px screen is 28px and should be 20px.
Also, why in additional info the title block is unique to additional info? why not use content title?

Reply:
switched